package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "itapuc-apprenticeship-project",
+  "version": "1.0.0",
+  "description": "Task 2",
+  "main": "index.js",
```

Comment:
you can go ahead an change this to `server.js` - although since we're not publishing this an an `npm` package this won't have any effect.  This field basically says, if I `npm install` this package, and import it with

```js
require("itapuc-apprenticeship-project");
```
or
```js
import "itapuc-apprenticeship-project";
```
what is the main file to enter?